lib/Exporter/Writer/FormatedBoolWriter.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Sonata package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Writer;
+
+/**
+ * Helper that format boolean in yes/no.
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+class FormatedBoolWriter implements WriterInterface
+{
+    protected $writer;
+
+    /**
+     * @param Exporter\WriterInterface $writer
+     */
+    public function __construct(WriterInterface $writer)
+    {
+        $this->writer = $writer;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function open()
+    {
+        $this->writer->open();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function close()
+    {
+        $this->writer->close();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function write(array $data)
+    {
+        foreach ($data as $key => $value) {
+            if (is_bool($data[$key])) {
+                $data[$key] = $data[$key] === true ? 'yes' : 'no';
```

Comment:
it will be great if the yes and no value can be customized in the constructor. Also can you add unit test ? Thanks.